rules.json

```diff
@@ -1,17 +1,170 @@
+
+         "(Rules about applications)",
+         "• Do NOT ask department recruiters, department command staff or staff members to review your application.",
+         "• Do NOT send an application to a department if you already have an application pending for that department.",
+         "• Do NOT lie or make false statements in applications, such as lying about your age or previous TXNRP experience/history.",
+
+            "Staff report: https://txnrp.com/r/staffReport",
+            "Player report: https://txnrp.com/r/report",
+            "Bug report: Link https://txnrp.com/r/bug"
+        ]
+            
+
+
+
+
+
+
+            
+
+    },
+    {
+        "title": "Discord Rules",
+        "isChild": false,
+        "rules": [
+            "Text and Voice Channel(s)",
+    "• Please try your best to help us to keep this Discord, a safe and friendly place to chat.",
+    "• You are NOT ALLOWED to use inappropriate language that includes racism, sexism, or anything that is extremely offensive to specific groups of people. This can be in your status message, profile picture, all of the text chats, as well as all of the public voice channels.",
+    "• People are allowed to use PG-13 Language in any public voice channels as long as (1) consist of 8 or less people and (2) everyone in that voice lobby is okay with it.",
+    "• Do NOT try to bypass any bot filters.",
+    "• Do NOT spam in any text channel.",
+    "• Do NOT post inappropriate pictures if ever given the permission to post links/pictures.",
+    "Respect",
+    "• Respect and treat everyone EQUALLY. This means to treat and respect both members and the staff team how you would like to be treated.",
+    "• Maturity must be used at ALL TIMES, even when a Staff Member is not in the channel.",
+    "• Obey all staff members. If you feel like you are being treated unfairly by a staff member, please fill out a Staff Report on our forums. Do not complain about it in public.",
+    "• Do not impersonate ANY other members on the Discord Server. This also applies to any known YouTubers/Celebrities. This falls under “Intentionally Deceiving,” and you can be punished for it.",
+    "• Do NOT tag (using @ to ping or send a notification to a specific member) any staff members unless you have a valid reason to do so. Also, do not tag any specific users repeatedly.",
+    "• It is under your discretion to share your own personal information with other members such as race, sex, sexual orientation, financial status, full names, addresses, IP’s, appearances, and voices. However, do NOT share others’ personal information, unless given permission.",
+    "• For the sake of everyone, it is only fair and respectful that TXNRP is an ENGLISH only network. Please refrain from speaking any other languages than English in any text/voice channels.",
+    "Advertising",
+    "• Do NOT have a social media link, IP, or another FiveM server in your discord status message.",
```

Comment:
@IceTheDev do we still plan to enforce this?

Reply:
don't know yet, still need to go through the rules and re-write them. A lot of them are confusing and some just don't make any sense.